core-web/libs/template-builder/src/lib/components/template-builder/components/add-widget/add-widget.component.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    HostListener,
+    Input
+} from '@angular/core';
+
+import { colIcon, rowIcon } from './utils/icons';
+
+import { WidgetType } from '../../models/models';
+
+const iconsMap = {
+    row: rowIcon,
+    col: colIcon
+};
+
+@Component({
+    selector: 'dotcms-add-widget',
+    templateUrl: './add-widget.component.html',
+    styleUrls: ['./add-widget.component.scss'],
+    standalone: true,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class AddWidgetComponent {
+    public isDragging = false;
+
+    @Input() label = 'Add Widget';
+    @Input() type: WidgetType = 'row';
```

Comment:
Maybe is more lose to pass the icon instead of the type? because this way we tightly couple to type.